app/ui/src/app/integration/integration-support.api.ts

```diff
@@ -0,0 +1,16 @@
+import { Endpoints } from '@syndesis/ui/platform';
+
+export const integrationSupportEndpoints: Endpoints = {
+  // TODO does this belong here really
+  metadata: '/connections/{connectionId}/actions/{actionId}',
+  // TODO should this go into the integration service
+  filterOptions: '/integrations/filters/options',
+  history: '/integrations/{id}/history',
+  deployments: '/integrations/{id}/deployments',
+  deployment: '/integrations/{id}/deployments/{version}',
+  pom: '/integration-support/generate/pom.xml',
+  export: '/integration-support/export.zip',
+  import: '/integration-support/import',
+  // TODO this path should be driven by config.json also does this belong here
+  javaInspection: '../../mapper/v1/java-inspections/{connectorId}/{type}.json',
```

Comment:
Have you double checked wether this path works. With our current setup, the `ApiHttpService` will prepend the `apiHost` and `apiEndpoint` details and `javaInspection` will eventually turn into a call to:

`https://syndesis.whatever.com/api/v1../../mapper/v1/java-inspections/{connectorId}/{type}.json`

Please note the lack of slash between `v1` and `../..`. Also, this is a bit ugly either way. Perhaps we should update the `setEndpointUrl` method to allow for custom `apiEndpoint` values (an options parameter perhaps?) so we can bind `mapperEndpoint` by hand.

For the time being let's focus on checking if this endpoint works like this (with no leading slash) and then refactor on a later stage.

Reply:
Yeah, I think that's a good idea, there's always a potential we'll have calls that don't fall into `/api/v1` or even have `/api/v1beta1` etc.  I'll add the `/` and actually have to adjust the code calling this anyways, so thanks for the heads up!

I kinda think BTW we might consolidate calls to the data mapper stuff into it's own service in a future refactor, WDYT?

Reply:
Just so we don't forget (I know I will) created https://github.com/syndesisio/syndesis/issues/1211

Reply:
I added my comments on #1211 and I also think we should update `setEndpointUrl` to automatically append that slash if missing, so the concatenated URL is legal